package.json

```diff
@@ -32,7 +33,7 @@
     "jquery-migrate": "3.3.0",
     "jquery-ui-dist": "1.12.0",
     "jquery-validation": "^1.17.0",
-    "lodash": "4.17.15",
+    "lodash": "4.17.19",
```

Comment:
I think this version will generate problems when they deploy, I think VIITE tried to do that, please stay alert.

Reply:
This is the versions currently used in VIITE, but I will check on devtest when deployed